src/Php83/Php83.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Polyfill\Php83;
+
+/**
+ * @author Ion Bazan <ion.bazan@gmail.com>
+ *
+ * @internal
+ */
+final class Php83
+{
+    public static function json_validate(string $json, int $depth = 512, int $flags = 0): bool
+    {
+        if ($flags !== 0 && $flags !== 1048576) {
```

Comment:
This magic number is value of `JSON_INVALID_UTF8_IGNORE` constant which is available from PHP 7.2+. Should we provide it in `symfony/polyfill-php72` and require it in this polyfill instead?

Reply:
We decided to not polyfill constants for JSON flags because we cannot polyfill the functionality that is controlled via those constants.

In our case, you can use the real `JSON_INVALID_UTF8_IGNORE` constant if it's defined. If it isn't defined, passing that magic number is not supported either because the `json_decode()` function that you're calling with that flag wouldn't know what to do with it.

```suggestion
        if ($flags !== 0 && !(defined('JSON_INVALID_UTF8_IGNORE') && JSON_INVALID_UTF8_IGNORE === $flags)) {
```